packages/phoenix/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+#![no_std]
+
+pub mod utils;
+
+pub mod error;
+
+// pub mod token_contract {
+//     soroban_sdk::contractimport!(
+//         file = "../../target/wasm32-unknown-unknown/release/soroban_token_contract.wasm"
+//     );
+// }
+//
+// pub mod stake_contract {
+//     soroban_sdk::contractimport!(
+//         file = "../../target/wasm32-unknown-unknown/release/phoenix_stake.wasm"
+//     );
+// }
+//
+// #[allow(clippy::too_many_arguments)]
+// pub mod lp_contract {
+//     soroban_sdk::contractimport!(
+//         file = "../../target/wasm32-unknown-unknown/release/phoenix_pair.wasm"
+//     );
+// }
```

Comment:
```suggestion

```

Reply:
keeping that so we can use it throughout the project